### ISSUE TITLE
 help!help!help!Test run failed

### ISSUE THREAD

**Issue by PanJinquan** [Joker316701882/Deep-Image-Matting#23]
I download your Github code and the corresponding model file：https://github.com/Joker316701882/Deep-Image-Matting，Then run：python test.py --alpha=./test_data/alpha/1.png --rgb=./test_data/RGB/1.png，But error in this procedure：saver.restore(sess,tf.train.latest_checkpoint('./model'))#PS：I have downloaded your model and put it in the "model" file.
Here errors：InvalidArgumentError (see above for traceback): No OpKernel was registered to support Op 'MaxPoolWithArgmax' with these attrs. Registered devices: [CPU], Registered kernels:
<no registered kernels>

[[Node: pool1 = MaxPoolWithArgmax[T=DT_FLOAT, Targmax=DT_INT64, ksize=[1, 2, 2, 1], padding="SAME", strides=[1, 2, 2, 1]](conv1_2)]]

**Comment by elzeru** [comment]
Just tried 1.8 CPU, works like a charm, and the upgrade is very easy to do. On my Debian:
pip3 install --upgrade tensorflow

**Comment by shliang0603** [comment]
@PanJinquan  Can you send me the pretrained model, I can't download it ,my email is :shliang0603@gmail.com    Thanks

**Comment by shliang0603** [comment]
@PanJinquan 哥哥，你有Deep-Image-Matting的数据集吗，可以发我一份吗，谢谢啦 ，我的email: shliang0603@gmail.com

**Comment by chenhaocs** [comment]
> I download your Github code and the corresponding model file：[https://github.com/Joker316701882/Deep-Image-Matting，Then](https://github.com/Joker316701882/Deep-Image-Matting%EF%BC%8CThen) run：python test.py --alpha=./test_data/alpha/1.png --rgb=./test_data/RGB/1.png，But error in this procedure：saver.restore(sess,tf.train.latest_checkpoint('./model'))#PS：I have downloaded your model and put it in the "model" file.
> Here errors：InvalidArgumentError (see above for traceback): No OpKernel was registered to support Op 'MaxPoolWithArgmax' with these attrs. Registered devices: [CPU], Registered kernels:
> [[Node: pool1 = MaxPoolWithArgmax[T=DT_FLOAT, Targmax=DT_INT64, ksize=[1, 2, 2, 1], padding="SAME", strides=[1, 2, 2, 1]](conv1_2)]]

@PanJinquan Hello, could you send me your pre-trained model and the training datasets?
Thanks a lot!
My email chanhal@outlook.com

**Comment by rainsun1** [comment]
could you ALSO send me your pre-trained model and the training datasets?
Thanks!
email: yglhit@sina.cn

**Comment by rainsun1** [comment]
@PanJinquan could you ALSO send me your pre-trained model and the training datasets?
Thanks!
email: yglhit@sina.cn

**Comment by ssxlconch** [comment]
> I download your Github code and the corresponding model file：[https://github.com/Joker316701882/Deep-Image-Matting，Then](https://github.com/Joker316701882/Deep-Image-Matting%EF%BC%8CThen) run：python test.py --alpha=./test_data/alpha/1.png --rgb=./test_data/RGB/1.png，But error in this procedure：saver.restore(sess,tf.train.latest_checkpoint('./model'))#PS：I have downloaded your model and put it in the "model" file.
> Here errors：InvalidArgumentError (see above for traceback): No OpKernel was registered to support Op 'MaxPoolWithArgmax' with these attrs. Registered devices: [CPU], Registered kernels:
> [[Node: pool1 = MaxPoolWithArgmax[T=DT_FLOAT, Targmax=DT_INT64, ksize=[1, 2, 2, 1], padding="SAME", strides=[1, 2, 2, 1]](conv1_2)]]

sorry for bothering,
would you please send me the pretrained-model?
my emai: 2741592109@qq.com
